en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -211,7 +211,7 @@ Once the vehicle has performed sufficient horizontal movement to make the yaw ob
 
 Data from GPS receivers can be blended using an algorithm that weights data based on reported accuracy (this works best if both receivers output data at the same rate and use the same accuracy).
 The mechanism also provides automatic failover if data from a receiver is lost (it allows, for example, a standard GPS to be used as a backup to a more accurate RTK receiver).
-This is controlled by the [EKF2_GPS_MASK](../advanced_config/parameter_reference.md#EKF2_GPS_MASK) parameter.
+This is controlled by the [EKF2_GPS_MASK](../advanced_config/parameter_reference.md#EKF2_GPS_MASK) parameter. 
```

Comment:
@bkueng EKF2_GPS_MASK also appears to have disappeared. Do you know what has replaced it? (and whether I can just plug that in here, or it needs bigger change?)

Reply:
I don't know about that one